sanitiser/_ids.js

```diff
@@ -1,19 +1,58 @@
 var _ = require('lodash'),
     check = require('check-types'),
-    types = require('../query/types');
+    type_mapping = require('../helper/type_mapping');
 
 var ID_DELIM = ':';
 
-// validate inputs, convert types and apply defaults
-// id generally looks like 'geoname:4163334' (type:id)
-// so, both type and id are required fields.
+// validate inputs, convert types and apply defaults id generally looks like
+// 'geonames:venue:4163334' (source:layer:id) so, all three are required
 
 var lengthError = 'invalid param \'ids\': length must be >0';
 
 var formatError = function(input) {
-  return 'id `' + input + 'is invalid: must be of the format type:id for ex: \'geoname:4163334\'';
+  return 'id `' + input + ' is invalid: must be of the format type:id for ex: \'geonames:venue:4163334\'';
```

Comment:
The message is not right. Says it "must be of the format __type:id__" but should be __source:layer:id__